experiments/ClimaEarth/components/land/climaland_integrated.jl

```diff
@@ -527,23 +528,17 @@ function FluxCalculator.compute_surface_fluxes!(
     atmos_sim::Interfacer.AtmosModelSimulation,
     _...,
 )
+    # We should change this to be on the boundary_space
```

Comment:
Could you open an issue for this please?

Reply:
Done!